content/applications/finance/accounting/getting_started/process_overview/supplier_bill.rst

```diff
@@ -1,144 +1,106 @@
 ===========================
-From Vendor Bill to Payment
+From vendor bill to payment
 ===========================
 
-Once vendor bills are registered in Odoo, you can easily pay vendors for
-the correct amount and at the right time (not too late, not too early;
-depending on your vendor policy). Odoo also offers reports to track your
-aged payable balances.
+Once vendor bills are registered in Odoo, you can easily pay vendors for the correct amount and at
+the right time.
 
-If you want to control vendor bills received from your vendors, you can
-use the Odoo Purchase application that allows you to control and
-pre-complete them automatically based on past purchase orders.
-
-From Vendor Bill to Payment
-===========================
-
-Record a new vendor bill
-------------------------
-
-When a vendor bill is received, you can record it from :menuselection:`Purchases --> Vendor Bills`
-in the Accounting application. As a shortcut,
-you can also use the **New Bill** feature on the accounting dashboard.
-
-.. image:: supplier_bill/vendor_bill05.png
-   :align: center
+.. seealso::
+   - Tutorial `Registering a vendor bill <https://www.odoo.com/slides/slide/registering-a-vendor-bill-1683?fullscreen=1>`_
```

Comment:
```suggestion
   - Tutorial `Registering a vendor bill <https://www.odoo.com/slides/slide/registering-a-vendor-bill-1683?fullscreen=1>`_
```
FYI the mergebot now has an automatic 100-character limit, which automatically fails the check if a line goes beyond that threshold. Idk if it applies to links, but if it does fail, this might be why.

Reply:
It doesn't fail as is, and doesn't render ok when moving it to the line, so I let it this way.